src/classes/SshConnectionSession.ts

```diff
@@ -0,0 +1,23 @@
+export class SshConnectionSession {
```

Comment:
This name is actually a bit overloaded, since SSH2 has the concept of a "connection" and a connection can have more than one "session" (see the `onSession` event handler defined in this class).

This class makes it seem like it will deal with the *session* but in fact it really is just intended to handle a connection for a given client.

Will think about names, but maybe something like `SshClientConnectionHandler`